tests/launchsecurity/sev.go

```diff
@@ -0,0 +1,40 @@
+package launchsecurity
+
+import (
+	expect "github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/framework/checks"
+	"kubevirt.io/kubevirt/tests/libvmi"
+)
+
+var _ = Describe("[sig-compute]AMD Secure Encrypted Virtualization (SEV)", func() {
+	BeforeEach(func() {
+		checks.SkipTestIfNoFeatureGate(virtconfig.LaunchSecurityGate)
+		checks.SkipTestIfNotSEVCapable()
+		tests.BeforeTestCleanup()
+	})
+
+	It("should start a SEV VM", func() {
+		vmi := libvmi.NewSEVFedora()
+		vmi = tests.RunVMIAndExpectLaunch(vmi, 240)
+
+		By("Expecting the VirtualMachineInstance console")
+		Expect(console.LoginToFedora(vmi)).To(Succeed())
+
+		By("Verifying that SEV is enabled in the guest")
+		err := console.SafeExpectBatch(vmi, []expect.Batcher{
+			&expect.BSnd{S: "\n"},
+			&expect.BExp{R: console.PromptExpression},
+			&expect.BSnd{S: "dmesg | grep --color=never SEV\n"},
+			&expect.BExp{R: "AMD Secure Encrypted Virtualization \\(SEV\\) active"},
```

Comment:
Nit: using a raw string (backticks) should avoid having to escape the escape character (backslash)

Reply:
I actually spent quite some time figuring out that my test was failing because of non-escaped backslash here :) Anyway, updated as suggested, maybe raw string will improve readability here.

Reply:
... and the funny thing is that in newer kernels this message has been changed to "AMD Memory Encryption Features active: SEV". So no need to escape anything when switching to Fedora35 :)

Reply:
Urg.. I assume there's no sys node or whatever that we could check instead of dmesg?

Reply:
AFAIK no.. not from the guest :( But I will double-check: previously I tested with an older kernel and Fedora 35 uses a newer one, perhaps smth has changed... but I would not be so optimistic about it.

Reply:
The kernel checks if SEV is active by reading the MSRs and verifying the proper bits. Unfortunately apart from that dmesg message I cannot see anything that may point to current feature status.